charts/timescaledb-single/generate_kustomization.sh

```diff
@@ -146,6 +139,17 @@ To preview the deployment of the secrets:
 __EOT__
 }
 
+test -d "${KUSTOMIZE_DIR}" \
+    && echo "The directory \"${KUSTOMIZE_DIR}\" already exists. This tool will not overwrite a current kustomization." \
+    && echo "Using existing secrets..." \
```

Comment:
Seems reasonable, however I'm wondering if we should at least inform the user
and give him/her the option to cancel the current script before just installing the secrets
directly.

(This could automatically install secrets from a highly secure k8s cluster into a development cluster)